HOW_WE_USE_GITHUB.md

```diff
@@ -238,17 +238,180 @@ please post details to the [Nucleus forums](https://community.anaconda.cloud/).
+This does not need to be completed consecutively. So, it is possible to complete a
+spike which has been time boxed for one day over the course of three or four days.
+```
+
+#### Task spikes
+
+Task spikes belong to an existing issue. Common use cases for these types of spikes is
+determining if a feature request is viable and if we want to pursue it further, or if a
+reported bug is worth addressing.
+
+When creating a task spike, do the following:
+
+- Under the ticket tasks, add a description of the spike and what needs to be done
+- Add a time box next to this task
+- If necessary, add further sub-tasks one level underneath the spike task
+
+Here's an example of what that looks like:
+
+- [ ] Create a proposed fix for this bug  (time box: _1 day_)
+  - [ ] Create a pull request with your proposed approach
+  - [ ] Justify and explain your approach on this pull request
+
+For the example above, the developer will create a prototype for a bug fix that will later
+be discussed with the project team. We have limited it to a one day time box to indicate that we
+do not want to spend any more time that on the proposal.
+
+If a prototype has not been drafted by the time the day limit has been reached, the developer
+working on it will write up as much information as they can about the bug report on the issue.
+Afterwards, they will work together with other devlopers to determine further development is
+warranted. If so, the time box will be redefined and reset.
+
+#### Issue spikes
+
+Issue spikes are much like task spikes except that they have enough work to warrant an entire
+issue to themselves. These are often created when planning new features or investigating new
+technologies that could benefit the project team.
+
+When creating an issue spike, do the following:
+
+- Add the "spike" label to the issue
+- Add a "Tasks List" section with a list of all applicable tasks for the spike.
+  For example, if we expect to create another issue as a result of this spike, that should be stated
+  there.
+
+Here's an example issue spike:
+
+---
+
+**Description**
+
+In order to better understand the needs of conda users, we intend to create a prototype for a new
+feature.
+
+**Time box:** 3 days
+
+**Task List**
+
+- [ ] Create a technical specification for this new feature
+- [ ] Create a pull request demonstrating how this feature works with an example use case of it
+- [ ] Present these findings at the upcoming conda community meeting
+
+---
+
+The above example has several tasks which may take considerable time to complete; therefore, it
+warrants its own separate issue. It is also important to note that because this is an issue for
+a prototype, the work is optional and can be discontinued at any time. Further work on it will
+be determined by feedback from the project team or conda community.
+
+Just like task spikes, issue spike time boxes can also be renegotiated and reset if the project
+team feels that further work is necessary or warranted.
```

Comment:
This is way too complicated for regular contributors and represents a serious barrier of entry, we should dumb this text down so it become easy to understand and is less bureaucratic.

Reply:
@jezdez, I believe you are bringing up a good point.

This document is only intended for those us who spend a majority of their working hours on conda projects (everything under https://github.com/conda). Regular contributors (those submitting bug-fixes or suggesting enhancements) will not be pressured to follow these guidelines. Regardless, these individuals will still benefit by having this information available as it helps them better understand the core development team's project management practices.

Therefore, I would very much prefer not having this text simplified in anyway.

Reply:
I've mentioned this in an Anaconda-internal chat as well: by making the document structured in this way, we're setting the expected mode of contribution to be “high barrier” but we need to keep it at “low barrier” (or if possible lowering it) to include everyone in the community and scale the ecosystem.

Maybe if this were just about the conda repo, or just for conda-build, it'd be fine, since project teams are free to self-govern and find their best practices. But this document here defines a specific style of agile development for the conda community infrastructure, and that's written too specific for that.

It's also wrong to say that this document would only for people "who spend a majority of their working hours on conda projects (everything under https://github.com/conda)" since
a) if it becomes a file in the conda organization repos, of course it will be seen as a requirement for everyone
b) not everyone working on projects under the conda organization are working in their working hours
c) the people providing feedback here (so far) are not representing the people working on projects under the conda organization

In short I think it's repeating the same mistakes of the past of defining community processes by fiat and not proposals.